suse_caasp

```diff
@@ -8,7 +8,7 @@
 # Modified:    2019 October 25
 #############################################################
 
-SVER=5.0.0
+SVER=4.5.0
```

Comment:
I don't see where else it's used than the description line below, so I am not sure about the requirement to mention the patch version. 

I am new to this, so I would naively just use `SVER=4.5` and bump it in major/minor versions, which are probably a good reason to revisit this (due to the insertion of new features in said versions).

Reply:
My search through the code found this is the only use.

Reply:
I'm fine with using major/minor versions e.g. `SVER=4.5`